datalad/distribution/drop.py

```diff
@@ -140,6 +140,21 @@ class Drop(Interface):
         ~/some/dataset$ datalad drop --recursive
 
     """
+    _examples_ = [
+        dict(text="Drop single file content",
+             code_py="drop('path/to/file')",
+             code_cmd="datalad drop <path/to/file>"),
```

Comment:
Style comment: If we are using non-placeholder syntax `'path/to/file'` for one API, we should stick with it for the other.